cmd/crank/beta/beta.go

```diff
@@ -20,12 +20,14 @@ limitations under the License.
 package beta
 
 import (
+	"github.com/crossplane/crossplane/cmd/crank/beta/describe"
 	"github.com/crossplane/crossplane/cmd/crank/beta/render"
 	"github.com/crossplane/crossplane/cmd/crank/beta/xpkg"
 )
 
 // Cmd contains beta commands.
 type Cmd struct {
-	XPKG   xpkg.Cmd   `cmd:"" help:"Manage packages."`
-	Render render.Cmd `cmd:"" help:"Render a claim or XR locally."`
+	XPKG     xpkg.Cmd     `cmd:"" help:"Manage packages."`
+	Render   render.Cmd   `cmd:"" help:"Render a claim or XR locally."`
+	Describe describe.Cmd `cmd:"" help:"Describe a Crossplane resource."`
```

Comment:
```suggestion
	Describe describe.Cmd `cmd:"" help:"Describe a Crossplane resource."`
	Render   render.Cmd   `cmd:"" help:"Render a claim or XR locally."`
	XPKG     xpkg.Cmd     `cmd:"" help:"Manage packages."`
```
Sort commands alphabetically.

Reply:
Yes, I'll have to rebase it on top of the "sort everything" PR